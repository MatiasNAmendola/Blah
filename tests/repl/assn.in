a: 5
a
b: -6
b
a: True
a
c: 2 *
3 - 5 * 5 + 15
c
a: 22 + 6 * -4
a
d: a * (b - c) + c
d

