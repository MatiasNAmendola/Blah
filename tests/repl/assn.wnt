5
-6
True
-4
-2
0
not a c string
False
100
not a c string
