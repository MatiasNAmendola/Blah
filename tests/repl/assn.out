5
-6
True
-4
-2
0
